File: src/main/java/com/paragon/api/event/world/entity/EntityPushEvent.java
package com.paragon.api.event.world.entity;

import me.wolfsurge.cerauno.event.CancellableEvent;
import net.minecraft.entity.Entity;

public class EntityPushEvent extends CancellableEvent {

    private final Entity entity;

    public EntityPushEvent(Entity entity) {
        this.entity = entity;
    }

    public Entity getEntity() {
        return this.entity;
    }

}

File: src/main/java/com/paragon/api/event/world/entity/EntityPushEvent.kt
package com.paragon.api.event.world.entity

import me.wolfsurge.cerauno.event.CancellableEvent
import net.minecraft.entity.Entity

/**
 * @author Surge, SooStrator1136
 */
class EntityPushEvent(val entity: Entity) : CancellableEvent()